### ISSUE TITLE
Atom 1.30.0-beta0:  Error: Cannot find module string_decoder

### ISSUE THREAD

**Issue by lgeiger** [atom/atom#17765]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

After updating to Atom 1.30.0-beta0, activation of the Github package fails due to a missing indirect dependency with the following error:

```
module.js:543 Uncaught (in promise) Error: Cannot find module '../node_modules/string_decoder/index.js'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/Applications/Atom Beta.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom Beta.app/Contents/Resources/app/src/module-cache.js:354:58)
    at Module.require (file:///Applications/Atom%20Beta.app/Contents/Resources/app.asar/static/index.js:40:43)
    at require (internal/module.js:11:18)
    at customRequire (/Applications/Atom Beta.app/Contents/Resources/app/static/<embedded>:96:26)
    at get_Decoder (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/split/index.js:12:37)
    at split (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/split/index.js:20:28)
    at /Applications/Atom Beta.app/Contents/Resources/app/node_modules/what-the-status/index.js:37:18
    at new Promise (<anonymous>)
    at parse (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/what-the-status/index.js:16:16)
    at Proxy.getStatusBundle (/Applications/Atom Beta.app/Contents/Resources/app/node_modules/github/lib/git-shell-out-strategy.js:602:58)
    at <anonymous>
```

### Steps to Reproduce

1. Install Atom 1.30.0-beta0 for macOS
2. Start Atom in a folder with a git repo
3. Open the dev tools and observe the above error

**Expected behavior:** Github package activates properly

**Actual behavior:**  Github package doesn't show current repo status and throws the above error in the dev console 

**Reproduces how often:** 100%

### Versions

```
$ atom-beta --version
Atom    : 1.30.0-beta0
Electron: 2.0.5
Chrome  : 61.0.3163.100
Node    : 8.9.3
```
### Additional Information

I'm posting it here since this is likely a issue with during generation of the snapshot builds.
Related issues #17125, #17481 and #17603

**Comment by rsese** [comment]
Thanks for the report @lgeiger :+1:, just noting that I can reproduce on macOS 10.12.6 with 1.30.0-beta0.

**Comment by daviwil** [comment]
Thanks @lgeiger!  This issue just doesn't want to go away...  We're looking into it!

**Comment by daviwil** [comment]
Hey @lgeiger, we just shipped [`1.30.0-beta1`](https://github.com/atom/atom/releases/tag/v1.30.0-beta1) which should resolve this issue for you.  I'll go ahead and close this issue for now but let us know if you continue to see this error!

**Comment by lgeiger** [comment]
Thanks for the fast response! Works like a charm 🎉
